debug: False
# Publish pose path for visualizaion ni RViz
pub_pose_path: True
# If True, test cases will be run. No ROS subscribers
run_test_cases: True
# Plot only test cases
plot: True
# Use 6DoF (position, velocity), instead of 9DoF (position, velocity, acceleration)
use_6dof_model: True

# Localization frame id. Pose trajectory is published  with respect to it
reference_frame_id: "map"

# Smooth control signal, to avoid large control jumps 
enable_control_smoothing: False
# Weight/penality on control input differences
smooth_input_weight: 0.1

# 3D velocity limits in m/s. Shoild be >= 0
# [x,y,z]
max_velocity: [10.0, 10.0, 10.0]

# 3D (net) acceleration limits in m/s/s. Should be >= 0
# [x,y,z]
max_acceleration: [5.0, 5.0, 5.0]

# 3D jerk limits in m/s/s/s. Should be >= 0
# [x,y,z]
max_jerk: [2.0, 2.0, 2.0]

# Number of MPC horizon steps, shoud be > 1
mpc_window: 40

# Optimization penalty on the states (Q)
state_weight: 5000

# Optimization penalty on the control inuputs (R)
input_weight: 0.1


# Smapling time in seconds
dt: 0.05

# Full path to CSV where MPC data is saved
save_mpc_data: True
output_csv_file: "/home/arrow/shared_volume/mpc_data.csv"